### ISSUE TITLE
[Telemetry] Recent Tabs

### ISSUE THREAD

**Issue by gabrielluong** [mozilla-mobile/fenix#19955]
## Description & Product Manager / Data Scientist User Story

As a Product Manager I want to know how often people interact with the homescreen to open the "last opened tab" or the tabs tray. 

- User expanding the recent section with "Show all"
- User opening the last open tab (from the homescreen)

## Acceptance Criteria
- [x] ENG files a [DS JIRA](https://jira.mozilla.com/projects/DO/issues/DO-228?filter=allopenissues) request outlining their methodology. https://mozilla-hub.atlassian.net/browse/DO-535
- [ ] DS sign off on instrumentation methodology addressing product questions.
- [ ] Event pings can be queried via re:dash
- [ ] Event pings can be queried via amplitude
- [ ] We are sending telemetry events for the actions listed in the requirements
- [ ] We have documented the telemetry
- [ ] We have asked a data steward to [review](https://github.com/mozilla/data-review/blob/master/request.md) the telemetry

**Comment by gabrielluong** [comment]
This is telemetry for the feature in https://github.com/mozilla-mobile/fenix/pull/19944

**Comment by Mugurell** [comment]
@vesta0 @amedyne For this telemetry (and the one from https://github.com/mozilla-mobile/fenix/issues/20393) we need to choose whether:
- we want a simple count of user actions or 
- we also need a timestamp for each such action.

I'm thinking the second scenario would offer more insight 
- into how users are actually using this new feature, eg: if immediately after opening the app they select a recent tab or 
- we could correlate these events with other app features, eg: can check the scenario in which home is tapped and then users are tapping "Jump back in -> Show all" instead of opening the tabs tray to see their tabs.

Thoughts?

**Comment by amedyne** [comment]
Tagging @csadilek here, too. For considering this as experiments are designed.

**Comment by vesta0** [comment]
@Mugurell we want to understand if putting such "shortcuts" on the homescreen is useful - so yes it would be helpful to know whether a user opened the "last tab" right after opening the app and landing on homescreen, or after going to homescreen from tabs tray/an open tab. Let's add a timestamp for each action.

**Comment by lobontiumira** [comment]
Verified as fixed on the 8/2 Nightly build 92.0a1 with GV 92.0a1, with Google Pixel.
Ping: 9abd6ff4-45ab-488a-bc28-cb067370ecf9
```
{
          "category": "recent_tabs",
          "name": "show_all_clicked",
          "timestamp": 3253
        }
```

```
 {
          "category": "recent_tabs",
          "name": "in_progress_media_tab_opened",
          "timestamp": 9000
        }
```
Ping: e4f1697d-8294-4000-be0a-b2b1e3f20a6c
```
{
          "category": "recent_tabs",
          "name": "recent_tab_opened",
          "timestamp": 6454
        }
```

[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/mira-2108020001)
